### PR TITLE
samizreal's comment change

### LINKED ISSUE #10: samizreal's checklist
- [x] Create an issue
- [x] Create a branch
- [x] Make changes to our file
- [x] Make commits
- [x] Make a pull request
- [x] Perform a peer review
- [x] Merge

### PR DESCRIPTION
**Pull request by samizreal**
Making changes to the `samizreal` _post file.

These changes are based on and closes #10

### CODE REVIEW

File: _posts/0000-01-05-samizreal.md [hunk 1]
```diff
@@ -7,3 +7,4 @@ title: "Welcome to our slide deck!"
 {: .center}
 
 Caption here
```
Comment (webbj62) [inline _posts/0000-01-05-samizreal.md]: Maybe you should cleanup the "Caption here"

**Review by webbj62 — CHANGES_REQUESTED** [review]
Please review my comments

File: _posts/0000-01-05-samizreal.md (hunk 1 above)
Comment (samizreal) [reply _posts/0000-01-05-samizreal.md]: Good catch, Thanks!

**Review by samizreal — COMMENTED** [review]
Am I good to go?

**Review by beardofedu — APPROVED** [review]

**Review by webbj62 — APPROVED** [review]
LGTM

**Pull request merged**